frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -35,7 +35,11 @@ export const validationSchema = yup.object().shape({
 });
 
 const hasDomain = (url: string, domain: string): boolean => {
-  return url.includes(`https://${domain}/`) || url.includes(`@${domain}:`);
+  return (
+    url.includes(`https://${domain}/`) ||
+    url.includes(`https://www.${domain}/`) ||
```

Comment:
I personally would prefer `url.startsWith()` here, because I think it that what we expect here, or? But it looks good anyway.

Reply:
Updated.